valeriyapetrova/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+# get avg and var from rainfall
+class Rainfall
+  def initialize
+    @user_input = user_input
+    @city_and_values = {}
+  end
+
+  def run_cli
+    return if @user_input == 'exit!'
+
+    if @user_input.empty?
+      puts 'Mistake!!!'
+    else
+      puts "Rainfall mean: #{mean}"
+      puts "Rainfall variance: #{variance}"
+    end
+
+    repeat
+  end
+
+  def take_data
+    data = File.new('take_data.txt', 'r:UTF-8')
+    @data = data.read
```

Comment:
```suggestion
    @data = File.new('take_data.txt', 'r:UTF-8').read
```

btw why do you need 'r:UTF-8' herer?